presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkSessionProperties.java

```diff
@@ -38,6 +39,7 @@
     public static final String STORAGE_BASED_BROADCAST_JOIN_WRITE_BUFFER_SIZE = "storage_based_broadcast_join_write_buffer_size";
     public static final String SPARK_BROADCAST_JOIN_MAX_MEMORY_OVERRIDE = "spark_broadcast_join_max_memory_override";
     public static final String SPARK_SPLIT_ASSIGNMENT_BATCH_SIZE = "spark_split_assignment_batch_size";
+    public static final String SPARK_MEMORY_REVOKING_THRESHOLD = "spark_memory_revoking_threshold";
```

Comment:
This property makes development convenient. Otherwise, it requires configuration change and code review.